xl2times/utils.py

```diff
@@ -53,13 +53,38 @@ def apply_composite_tag(table: datatypes.EmbeddedXlTable) -> datatypes.EmbeddedX
         and table tag simplified.
     """
     if table.defaults:
-        varname = table.defaults
+        defaults = table.defaults
         df = table.dataframe
-        if "attribute" in df.columns:
-            df["attribute"] = df["attribute"].fillna(varname)
+        # Check for ANSWER-style defaults
+        if "=" in defaults:
+            # Split multiple comma-separated defaults
+            if "," in defaults:
+                defaults = defaults.split(",")
+            else:
+                # Make it a list
+                defaults = list([defaults])
```

Comment:
```suggestion
                defaults = [defaults]
```

Reply:
Removed the `if else` statemement, because `str.split()` returns a list.